lomap/graphgen.py

```diff
@@ -166,6 +172,31 @@ def __init__(self, dbase):
 
         return
 
+    #sliu 07/17 add function to pick the lead compounds for the radial graphing option
+    def pick_lead(self):
+        if (self.dbase.nums() * (self.dbase.nums() - 1)/2) != self.dbase.strict_mtx.size:
+            raise ValueError("There are errors in the similarity score matrices")
+        if not self.dbase.options.bias == "None":
+            #bias radial option
```

Comment:
This would be a good place to recap what this does as opposed to just naming it (and likewise on line 189 in the same file for "complete radial option". Maybe here you could say "Use the provided reference compound as a hub" and below you could say, "Pick the compound with the highest total similarity to all other compounds to use as a hub" or something...?